apps/io-profile-async/src/repositories/session-notifications.ts

```diff
@@ -66,8 +71,30 @@ export const updateNotificationEvents: (
     notificationEvents
   });
 
+// TODO: add tests
+const updateExpiredSessionNotificationFlag: (
+  actualRecord: RetrievedSessionNotifications,
+  flagNewValue: boolean
+) => RTE.ReaderTaskEither<Dependencies, CosmosErrors, void> = (
+  actualRecord: RetrievedSessionNotifications,
+  flagNewValue: boolean
+) => (deps: Dependencies) =>
```

Comment:
There’s no need to use `RetrievedSessionNotifications`, nor to redefine the input types.
We might even consider passing only the `fiscalCode` and `expiredAt` values.

```suggestion
const updateExpiredSessionNotificationFlag: (
  actualRecord: SessionNotifications,
  flagNewValue: boolean
) => RTE.ReaderTaskEither<Dependencies, CosmosErrors, void> = (
  actualRecord,
  flagNewValue
) => (deps: Dependencies) =>
```

Reply:
2146fe07ea48eebca39d522c1c9e4ac1a5dc52ed